File: jdroid-gradle-commons-plugin/src/main/java/com/jdroid/gradle/commons/JavaBaseGradleExtension.java
package com.jdroid.gradle.commons;

import org.gradle.api.Project;

public class JavaBaseGradleExtension extends BaseGradleExtension {
	
	private String integrationTestsPattern = "**/integration/**/*Test.class";

	public JavaBaseGradleExtension() {
		// TODO Remove this constructor. Only for testing
		super();
	}

	public JavaBaseGradleExtension(Project project) {
		super(project);
	}
	
	public String getIntegrationTestsPattern() {
		return integrationTestsPattern;
	}

	public void setIntegrationTestsPattern(String integrationTestsPattern) {
		this.integrationTestsPattern = integrationTestsPattern;
	}
}

File: jdroid-gradle-commons-plugin/src/main/groovy/com/jdroid/gradle/commons/JavaBaseGradleExtension.kt
package com.jdroid.gradle.commons

import org.gradle.api.Project

open class JavaBaseGradleExtension : BaseGradleExtension {

    var integrationTestsPattern = "**/integration/**/*Test.class"

    // TODO Remove this constructor. Only for testing
    constructor() : super()

    constructor(project: Project) : super(project)
}
